src/Html.php

```diff
@@ -1147,8 +1147,8 @@ public static function includeHeader(
         $tpl_vars['css_files'][] = ['path' => 'public/lib/photoswipe.css'];
         Html::requireJs('photoswipe');
 
-       //on demand JS
-        if ($sector != 'none' || $item != 'none' || $option != '') {
+        //on demand JS. Always evaluated when debug mode is enabled so requirements for the debug bar can be loaded.
+        if ($sector != 'none' || $item != 'none' || $option != '' || $_SESSION['glpi_use_mode'] === Session::DEBUG_MODE) {
```

Comment:
The following piece of code could also be added before this `if` block:
```php
            if ($_SESSION['glpi_use_mode'] === Session::DEBUG_MODE) {
                $tpl_vars['css_files'][] = ['path' => 'public/lib/codemirror.css'];
                Html::requireJs('codemirror');
            }
```

and then it would probably just require a `$tpl_vars['css_files'] = array_unique($tpl_vars['css_files']);` call after the `if` block to prevent loading the `codemirror` CSS twice.